Apps/phradar/radar.json

```diff
@@ -10,12 +10,13 @@
     "product_name": "Radar",
     "product_version_regex": ".*",
     "publisher": "RADAR, LLC",
-    "license": "Copyright (c) 2020 Splunk Inc.",
+    "license": "Copyright (c) 2020-2021 Splunk Inc.",
```

Comment:
@pauwlsky As per the new standard practice, the copyright license should be third-party as the publisher value is third-party. So, if you can provide the license value in the format "Copyright (c) {year} {license}", it would be a help to us and we will use it in all the files to mention the copyright license.

Reply:
@mjiyani-crest `Copyright (c) 2021 RADAR, LLC` would work (or with the year range 2020-2021, that is also fine).